dwapi/webhooks.go

```diff
@@ -33,27 +34,35 @@ func (s *WebhookService) List() (response []Subscription, err error) {
 	return
 }
 
+// RetrieveAccountSubscription fetches the webhook subscription associated with the currently
+// authenticated user and to the given organization or user account.
```

Comment:
>webhook subscription associated with the currently authenticated user and to the given...

I think this sentence fragment could be more clear -- the mix of prepositions (`with` and `to`, specifically) tripped me up.

Maybe "webhook subscription based on the currently authenticated user and the given..."?